ansible/roles/stack-sunbird/templates/sunbird_cert-registry-service.env

```diff
@@ -6,11 +6,13 @@ sunbird_es_port={{sunbird_es_port}}
 sunbird_cassandra_keyspace={{sunbird_cassandra_keyspace|d('sunbird')}}
 sunbird_cassandra_username={{sunbird_cassandra_username|d('cassandra')}}
 sunbird_cassandra_password={{sunbird_cassandra_password|d('password')}}
-sunbird_cassandra_consistency_level=quorum
+sunbird_cassandra_consistency_level={{sunbird_cassandra_consistency_level}}
```

Comment:
Add a default value in `all.yml` for `sunbird_cassandra_consistency_level`

Reply:
@keshavprasadms All reay present in all.yml. But value is "one" , shall we change it to "quorum"? when multi dc is enabled it has to change to "local_quorum"

Reply:
We can use `local_quorum` for single dc also.